Formula/amqp-cpp.rb

```diff
@@ -0,0 +1,30 @@
+class AmqpCpp < Formula
+  desc "C++ library for communicating with a RabbitMQ message broker"
+  homepage "https://github.com/CopernicaMarketingSoftware/AMQP-CPP"
+
+  stable do
```

Comment:
Don't need a `stable do` block unless you're doing something unique with stable, such as a dependency or resource not needed by `head`, etc.